internal/config/profile.go

```diff
@@ -243,6 +244,10 @@ func (p *Profile) Name() string {
 
 func SetName(name string) { Default().SetName(name) }
 func (p *Profile) SetName(name string) {
+	if strings.Contains(name, ".") {
+		log.Fatal("Profile should not contain '.'")
+	}
+
```

Comment:
Could be a follow-up ticket, but worth enquiring whether there are other special characters we should account for

Reply:
I don't think there's any other special characters we want to block for now.
The main reason we're blocking dots is because they trigger nested keys in viper.
Disallowing dots fixes that issue.

See: https://github.com/spf13/viper#accessing-nested-keys